services/src/test/scala/cromwell/services/database/DatabaseSystem.scala

```diff
@@ -13,13 +13,13 @@ sealed trait DatabaseSystem {
 
 object DatabaseSystem {
   val All: Seq[DatabaseSystem] = List(
-    HsqldbDatabaseSystem,
-    MariadbEarliestDatabaseSystem,
-    MariadbLatestDatabaseSystem,
-    MysqlEarliestDatabaseSystem,
-    MysqlLatestDatabaseSystem,
+//    HsqldbDatabaseSystem,
+//    MariadbEarliestDatabaseSystem,
+//    MariadbLatestDatabaseSystem,
+//    MysqlEarliestDatabaseSystem,
+//    MysqlLatestDatabaseSystem,
```

Comment:
Pretty sure we don't want this change to be immortalized...

Reply:
omg no